### DISCUSSION TITLE
Current cost wrong

### DISCUSSION THREAD

**Discussion by stephencrowther** [BottlecapDave/HomeAssistant-OctopusEnergy#1091] [Q&A]
Not sure if something I’m doing wrong but is it correct that the current accumulative cost sensor can’t handle negative values following some plunge/negative pricing.

currently my sensor shows 0.1, when in the official octopus app it shows as -0.40 (-40 pence)

**Comment by bazcurtis** [comment]
I was also wondering about this. I am currently force discharging my battery and the cost is not going down.

**Reply by stephencrowther** [reply]
After posting this I looked in developer tools and found an entity that was showing the right figure - cost tracker.

not sure why there is a difference between this and the current accumulative cost entity though! Maybe some reason why the two are calculated differently?

**Reply by BottlecapDave** [reply]
The current accumulative cost sensor is on a per meter basis (assuming you're refering to https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy/entities/electricity/#current-accumulative-cost), so the import sensor will only ever be positive. Unfortunately the octopus mini only reports import data and doesnt provide export information.

The previous accumulative cost sensor (https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy/entities/electricity/#previous-accumulative-cost) behaves in a similar way, but supports export meters. You could use a template sensor to combine both the import and export sensors together to get the overall cost.

The coat tracker is for something different - https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy/setup/cost_tracker/
